tests/filtered_relation/tests.py

```diff
@@ -248,10 +255,11 @@ def test_difference(self):
         self.assertSequenceEqual(qs1.difference(qs2), [self.author1])
 
     def test_select_for_update(self):
+        qs = Author.objects.annotate(
+            book_jane=FilteredRelation('book', condition=Q(book__title__iexact='the book by jane a')),
+        ).filter(book_jane__isnull=False).select_for_update()
         self.assertSequenceEqual(
-            Author.objects.annotate(
-                book_jane=FilteredRelation('book', condition=Q(book__title__iexact='the book by jane a')),
-            ).filter(book_jane__isnull=False).select_for_update(),
+            qs,
```

Comment:
This change is unrelated, please revert it.

Reply:
addressed